client/landing/gutenboarding/onboarding-block/question/index.tsx

```diff
@@ -19,16 +20,18 @@ interface Props {
 /* eslint-disable wpcalypso/jsx-classname-namespace */
 
 const Question: FunctionComponent< Props > = ( {
-	isActive,
-	onExpand,
+	children,
+	className,
 	displayValue,
+	isActive,
 	label,
-	children,
+	onExpand,
 } ) => (
 	<div
-		className={ classNames( 'onboarding-block__question', {
+		className={ classNames( 'onboarding-block__question', className, {
 			selected: isActive,
 		} ) }
+		onClick={ () => void ( ! isActive && onExpand() ) }
```

Comment:
I don't think we are supposed to make the whole question clickable (including the black text). 
At least by going through design notes I see this:
> Clicking the blue links in previous steps (ex. “Setting up a site for a business”) returns to that step, with all input saved for other completed steps.

Reply:
Good spot, making the entire madlib clickable seemed most comfortable to me but the doc seems to indicate otherwise

@dwolfe @olesyabrk would you prefer (A) _only_ the fillable part of the madlibs in capture intent focus the input, or (B) should the non-editable text also focus the input?

A (click editable text):

![a](https://user-images.githubusercontent.com/841763/68936236-663f3d80-079a-11ea-8770-c5b03f646f16.gif)


B (click any part of madlib):

![b](https://user-images.githubusercontent.com/841763/68936256-71926900-079a-11ea-8f4a-199c2e8f0576.gif)

Reply:
I think the way you have it now makes the most sense, given the current design (ie. option A, only the editable text is actionable). I get why you're asking - in the radio button example, obviously the label provides a larger hit area than just the radio itself - but the small text is 36px tall x [length of whatever the user types in, which will almost always be > 36px], so I feel like that's a large enough target.

I don't have a strong opinion either way, but if I had to choose, I'd say Option A is just fine.

Reply:
Thanks! Will go with option **A** 👍